infra/discovery/src/lib/search.js

```diff
@@ -342,7 +420,13 @@ class Listing {
         subCategory: hit._source.subCategory,
         description: hit._source.description,
         priceAmount: (hit._source.price || {}).amount,
-        priceCurrency: (hit._source.price || {}).currency
+        priceCurrency: (hit._source.price || {}).currency,
```

Comment:
I would think so, but that actually breaks it if I do that. Not totally sure why. This code was originally here before I started so I'm following convention.

Reply:
The data in ElasticSearch roughly comes from our GraphQL data schema. On a listing, "price" is a Price object. Here's some definitions from the typeDefs/Common.js

```
  type Price {
    currency: CurrencyResult
    amount: String
  }
...
  interface Currency {
    id: ID!
    name: String
    code: String
    priceInUSD: Float
  }
```

So Price is an object. I've got the `|| {})` in there to prevent crashing the request getting the amount if for some reason the price object isn't there.